src/components/WishlistItems/WishlistItems.jsx

```diff
@@ -0,0 +1,53 @@
+import { useWishlist } from "../../context/index";
+import { removeFromWishlist } from "../../ApiCalls/index";
+
+export const WishlistItems = () => {
+  const { wishlist, wishlistDispatch } = useWishlist();
+
+  return (
+    <>
+      {wishlist.length === 0 ? (
+        <h1>No products added</h1>
+      ) : (
+        wishlist.map((wishlistItem) => (
+          <div class="vertical-card-container card-hover">
+            <img class="card-image" src={wishlistItem.img} />
+
+            <div class="card-content">
+              <h6 class="card-title">{wishlistItem.title}</h6>
+              {/* <p class="card-info p-sm">Nitori snack bowls that may be...</p> */}
```

Comment:
Is this comment serve a purpose? if not better to remove it

Reply:
> Code looks neat and readable. A suggestion I have is about the naming of files and folders. Generally, all folder names and all file names except fo JSX components should be in lowercase seperated by `-`. eg: `api-calls` instead of `ApiCalls`.

Thank you for pointing out I have made the changes and also changed file name.